src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/CompanyConfirmationPageController.java

```diff
@@ -79,28 +82,34 @@ public String postCompanyInformation(@PathVariable("companyNumber") String compa
         eacRequest.setCompanyNumber(companyNumber);
 
         try {
-            CompanyDetail companyDetail = getCompanyDetails(companyNumber);
-            if( !ACCEPTED_STATUS.equals(companyDetail.getCompanyStatus()) || !ACCEPTED_TYPES.contains(companyDetail.getType())){
-                return UrlBasedViewResolver.REDIRECT_URL_PREFIX + "/auth-code-requests/company/" + companyNumber + CANNOT_USE_THIS_SERVICE;
+            CompanyDetail companyDetail = companyService.getCompanyDetail(companyNumber);
+
+            if(StringUtils.isEmpty(companyDetail.getDateOfCreation())) {
```

Comment:
Just double checked with our PO and she said having an empty incorporation date should not count as a disqualifier for the service, so I don’t think we need this check

Reply:
Ok, thanks @rsachdevch - I've amended the code 👍